charts/values.yaml

```diff
@@ -0,0 +1,250 @@
+## Default values for charts.
+## This is a YAML-formatted file.
+## Declare variables to be passed into your templates.
+
+## @param global clusterpedia global config
+global:
+  ## @param dbInstallMode "internal" and "external" are provided
+  ## "internal" means automatic Database installation, default mysql
+  ## "external" means to install the database yourself, must specify the database connection parameters
+  storageInstallMode: "internal"
+  ## persistence configuration
+  ##
+  persistence:
+    ## @param global.persistence.matchNode define the nodeAffinity of pv.
+    ## if defined, will create the pv on the node
+    matchNode: kind-control-plane
+  ## @param global.postgresql.auth define the auth info.
+  postgresql:
+    auth:
+      ## @param auth.postgresPassword Password for the "postgres" admin user
+      ##
+      postgresPassword: "dangerous"
```

Comment:
Overriding `global.postgres` doesn't seem to make sense, `postgresql` will be overridden below